wasm/src/main/java/com/dylibso/chicory/wasm/types/CodeSection.java

```diff
@@ -22,4 +30,28 @@ public int functionBodyCount() {
     public FunctionBody getFunctionBody(int idx) {
         return functionBodies.get(idx);
     }
+
+    public static final class Builder {
```

Comment:
It's somewhat surprising that the builders don't have a type hierarchy which mirrors their section type hierarchy. A common `Builder` supertype would be a good place to put a `parse` method, to make parsing more uniform.